src/app/form/(formSteps)/personal-information/page.tsx

```diff
@@ -1,11 +1,460 @@
+  lastName: string | null;
+  dateOfBirthMonth: string | null;
+  dateOfBirthDay: string | null;
+  dateOfBirthYear: string | null;
+  streetAddress1: string | null;
+  streetAddress2: string | null;
+  city: string | null;
+  state: string | null;
+  zip: string | null;
+}
+
+const orderedInputNameToLabel: { [key in keyof PersonalInformationData]: string } = {
+  juiceShopExperience: "Juice shop experience",
+  firstName: "First name",
+  middleName: "Middle name",
+  lastName: "Last name",
+  dateOfBirthMonth: "Month",
+  dateOfBirthDay: "Day",
+  dateOfBirthYear: "Year",
+  streetAddress1: "Street address 1",
+  streetAddress2: "Street address line 2",
+  city: "City",
+  state: "State",
+  zip: "ZIP code",
+};
+const mailingAddressHeader = "Mailing address";
 
 const PersonalInformationStep: React.FC = () => {
+  const router = useRouter();
+  const formProgressPosition = useFormProgressPosition();
+  const [dataHasLoaded, setDataHasLoaded] = useState<boolean>(false);
+  const [shouldSummarizeErrors, setShouldSummarizeErrors] = useState(false);
+  const {
+    register,
+    formState: { errors },
+    handleSubmit,
+    setFocus,
+    watch,
+  } = useForm<PersonalInformationData>({
+    defaultValues: {
+      juiceShopExperience: "",
+      firstName: "",
+      middleName: "",
+      lastName: "",
+      dateOfBirthMonth: "",
+      dateOfBirthDay: "",
+      dateOfBirthYear: "",
+    },
+    shouldFocusError: false,
+  });
+  const errorSummaryRef = useRef<HTMLInputElement>(null);
+  const zip = watch("zip");
+
+  const onSubmit: SubmitHandler<PersonalInformationData> = (data) => {
+    let key: keyof PersonalInformationData;
+    for (key in data) {
+      const value = data[key] ?? "";
+      setKeyValue(key, value);
+    }
+    routeToNextStep(router, formProgressPosition);
+  };
+  const onError: SubmitErrorHandler<PersonalInformationData> = (errors) => {
+    if (Object.keys(errors).length >= 3) {
+      setShouldSummarizeErrors(true);
+    } else {
+      setShouldSummarizeErrors(false);
+      for (const inputName of Object.keys(orderedInputNameToLabel) as Array<
+        keyof PersonalInformationData
+      >) {
+        if (errors[inputName] !== undefined) {
+          break;
+        }
+      }
+    }
+  };
+
+  useEffect(() => {
+    setDataHasLoaded(true);
+  }, []);
+
   return (
     <div>
-      <form className="usa-form">personal information goes here</form>
+      {dataHasLoaded && (
+        <Form onSubmit={handleSubmit(onSubmit, onError)} className="maxw-full" noValidate>
+          <div className="grid-row grid-gap-lg">
+            <div className="flex-2">
+              <div tabIndex={-1} ref={errorSummaryRef}>
+                {shouldSummarizeErrors && Object.keys(errors).length >= 1 && (
+                  <div
+                    className="usa-alert usa-alert--error margin-bottom-3 border-05 border-top-105 border-secondary-dark"
+                    aria-labelledby="error-summary-heading"
+                  >
```

Comment:
PEBBLE - This will not get read by the screen reader. I would recommend adding:

`role="alert"`

to the top level div to ensure the content is read by the screen reader.

Note: this will not be read if the content within the div does not change. Out of scope of exercise